protected_branches.go

```diff
@@ -187,6 +187,34 @@ func (s *ProtectedBranchesService) UnprotectRepositoryBranches(pid interface{},
 	return s.client.Do(req, nil)
 }
 
+// AllowForcePushOptions represents the available
+// AllowForcePush() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/protected_branches.html#allow_force_push # FIXME: not documented yet but this is what is done by the UI
+type AllowForcePushOptions struct {
```

Comment:
This could probably be merged with `RequireCodeOwnerApprovals()` into a generic `UpdateProtectedBranch()` function.